lib/ex_aws/dynamo.ex

```diff
@@ -115,6 +117,17 @@ defmodule ExAws.Dynamo do
           | :provisioned
   @type key_schema :: [{atom | binary, :hash | :range}, ...]
   @type key_definitions :: [{atom | binary, dynamo_type_names}, ...]
+  @type create_table_opts :: [
+          {:global_indexes, [map()]},
+          {:local_indexes, [map()]},
+          {:read_capacity, pos_integer},
+          {:write_capacity, pos_integer},
+          {:billing_mode, dynamo_billing_types},
+          {:stream_enabled, boolean()},
+          {:stream_view_type, stream_view_type()}
+        ]
+  @type stream_view_type :: :keys_only | :new_image | :old_image | :new_and_old_images
+  @type update_table_opts :: create_table_opts()
```

Comment:
I intentionally made them different so that if, in the future, there were create- or update-specific opts, it wouldn't break anyone's typespecs (assuming they'd used the right one).

Reply:
Alright by me.